python-api/gstswitch/server.py

```diff
@@ -22,6 +26,110 @@
 TOOLS_DIR = '/'.join(os.getcwd().split('/')[:-1]) + '/tools/'
 
 
+class ProcessMonitor(subprocess.Popen):
+    """Runs a Command in a Background-Thread and monitors it's output
+
+    Can block until the Command prints a certain string and log the full
+    output into a file
+    """
+
+    def __init__(self, cmd, logtarget=sys.stderr):
+        self.log = logging.getLogger('server-output-monitor')
+
+        # Logfile to write to
+        self._logtarget = logtarget
```

Comment:
`self._logtarget` isn't some special thing right - it is just where we copy the command's output too, right? 

Maybe `self._output_target` (or maybe `self._cmd_output_target`) might be a better name so people don't think the self.log commands are sending their output to this value too?